site/static/scripts/sign-android

```diff
@@ -153,13 +153,16 @@ if [ "$PACKAGE_ID" != "" ]; then
     if [ "$GOOGLE_SERVICES_PATH" != "" ]; then
         echo "Replacing Google Services values"
         GOOGLE_SERVICES=$(jq . "$GOOGLE_SERVICES_PATH")
+        PROJECT_ID=$(echo "$GOOGLE_SERVICES" | jq -r .project_info.project_id)
         PROJECT_NUMBER=$(echo "$GOOGLE_SERVICES" | jq -r .project_info.project_number)
-        FIREBASE_URL=$(echo "$GOOGLE_SERVICES" | jq -r .project_info.firebase_url)
+        FIREBASE_URL="https://$PROJECT_ID.firebaseio.com"
+        echo "Setting  FIREBASE_URL to: $FIREBASE_URL"
```

Comment:
@sadohert maybe remove this echo ?

Reply:
I left the echo in as a bit of a breadcrumb in case the logic picked something up incorrectly.